ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -580,7 +582,11 @@ public Collection<CommitInfoProto> getCommitInfos() {
       role.getLeaderState().ifPresent(
           leader -> leader.updateFollowerCommitInfos(commitInfoCache, infos));
     } else {
-      getRaftConf().getAllPeers().stream()
+      StreamSupport.stream(
+              Iterables
```

Comment:
@qian0817 , thanks a lot for working on this!

Let's use `Stream.concat` and also the same conf (otherwise, it could be changed.)
```java
      final RaftConfigurationImpl conf = getRaftConf();
      Stream.concat(conf.getAllPeers().stream(), conf.getAllPeers(RaftPeerRole.LISTENER).stream())
```